docs/reference/api/widgets/canvas.rst

```diff
@@ -105,21 +105,52 @@ Notes
 Reference
 ---------
 
-Main Interface
+Main interface
 ^^^^^^^^^^^^^^
 
 .. autoclass:: toga.Canvas
-.. autoclass:: toga.widgets.canvas.Context
-
-Simple Drawing objects
-^^^^^^^^^^^^^^^^^^^^^^
-
-.. automodule:: toga.widgets.canvas
-   :exclude-members: Canvas, Context, ClosedPathContext, FillContext, StrokeContext
+    :exclude-members: new_path, move_to, line_to, bezier_curve_to, quadratic_curve_to,
+        arc, ellipse, rect, write_text, rotate, scale, translate, reset_transform,
+        closed_path, fill, stroke
```

Comment:
I know we normally include deprecated methods in the documentation, but in this case there are so many of them that they would make it impossible to see an overview of the non-deprecated Canvas API. It's not critical to document them, because anyone who's using them will get a DeprecationWarning telling them exactly what to do.